### PR TITLE
updated batch run with compatible to unstract-client V0.2.0

### PR DESCRIPTION
**Pull request by muhammad-ali-e**
- Bumped `unstract-client` to version `0.2.0` to incorporate new features and improvements.
- Added the `include_metadata` option, allowing users to include additional metadata in the result for each file processed by `unstract-client`.

## Related Dependency:
-  See dependent PR for additional context and fixes: [unstract-python-client#7](https://github.com/Zipstack/unstract-python-client/pull/7).

## Related Issues
- https://github.com/Zipstack/unstract-api-deployments-batch-run/issues/5

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -1,3 +1,3 @@
-unstract-client~=0.1.0
+unstract-client~=0.2.0
```
Comment (ritwik-g) [inline requirements.txt]: @muhammad-ali-e shouldn't this had been 
```suggestion
unstract-client~=1.0.0
```

**Review by ritwik-g — APPROVED** [review]

File: README.md [hunk 2]
```diff
@@ -60,6 +60,7 @@ This will display detailed usage information.
 - `--skip_unprocessed`: Skip unprocessed files when retrying failed files.
 - `--log_level`: Log level (default: `INFO`).
 - `--print_report`: Print a detailed report of all processed files at the end.
+- `--include_metadata`: Include metadata in the result for each file.
```
Comment (chandrasekharan-zipstack) [inline README.md]:
```suggestion
- `--include_metadata`: Include metadata on tokens consumed and the context passed to LLMs for prompt studio exported tools in the result for each file.
```

File: main.py [hunk 3]
```diff
@@ -409,6 +411,13 @@ def main():
         help="Print a detailed report of all file processed.",
     )
 
+    parser.add_argument(
+        "--include_metadata",
+        dest="include_metadata",
+        action="store_true",
+        help="Include metadata in the result for each file.",
```
Comment (chandrasekharan-zipstack) [inline main.py]:
```suggestion
        help="Include metadata on tokens consumed and the context passed to LLMs for prompt studio exported tools in the result for each file.",
```

**Review by chandrasekharan-zipstack — APPROVED** [review]
LGTM

**Review by jaseemjaskp — APPROVED** [review]
LGTM

**Pull request merged**